pom.xml

```diff
@@ -13,6 +13,7 @@
   <properties>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
+    <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
```

Comment:
This is what was making  prints I added to a test go to a file when I running this locally. Was wondering what was doing that.